src/features/processPicker.ts

```diff
@@ -246,4 +418,27 @@ function execChildProcess(process: string, workingDirectory: string): Promise<st
             resolve(stdout);
         });
     });
+}
+
+function execChildProcessAndOutputErrorToChannel(process: string, workingDirectory: string, channel: vscode.OutputChannel): Promise<string> {
+    return new Promise<string>((resolve, reject) => {
+        child_process.exec(process, { cwd: workingDirectory, maxBuffer: 500 * 1024 }, (error: Error, stdout: string, stderr: string) => {
+            if (error) {
+                reject(new Error("See remote-attach output."));
+                channel.append(error.message);
+                channel.show();
+                return;
+            }
+
+            if (stderr && stderr.length > 0) {
+                reject(new Error("See remote-attach output"));
```

Comment:
I'm confused about how to do this. I thought this would be part of the callback function after it finishes the exec.